src/main/java/me/arminb/sara/services/UserServiceImpl.java

```diff
@@ -0,0 +1,41 @@
+package me.arminb.sara.services;
+
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+import me.arminb.sara.entities.User;
+import org.bson.Document;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import me.arminb.sara.dao.UserDAO;
+import me.arminb.sara.entities.User;
+
+
+@Service("userService")
+public class UserServiceImpl implements UserService {
+
+    @Autowired
+    private UserDAO userDAO;
+
+    @Autowired
+    private MongoDatabase database;
+
+    @Override
+    public void findAll() {
```

Comment:
return values should not be null. the same comments in the DAO implementation applies here.